src/schemas/28-1900/schema.js

```diff
@@ -42,7 +42,15 @@ const schema = {
     },
     yearsOfEducation: {
       type: 'string',
-      pattern: '^\\d+$',
+      enum: ['10', '12', '14', '15', '17', '19'],
+      enumNames: [
+        'Some high school',
+        'High school / GED',
+        'Some college',
+        'Associate degree',
```

Comment:
```suggestion
        'Associate's degree',
```

Reply:
These options were given to us by Platform Content - https://github.com/department-of-veterans-affairs/va.gov-team/issues/23998#issuecomment-829470556